library/cmac.c

```diff
@@ -868,8 +869,9 @@ static int cmac_test_wth_cipher( int verbose,
              * the underlying primitive itself is implemented alternatively,
              * certain features (e.g. AES-192) may be unavailable. This should
              * not cause the selftest function to fail. */
-            if( ret == MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED ||
-                ret == MBEDTLS_ERR_CIPHER_FEATURE_UNAVAILABLE ) {
+            if( ( ret == MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED
+                  || ret == MBEDTLS_ERR_CIPHER_FEATURE_UNAVAILABLE )
+                && cipher_type == MBEDTLS_CIPHER_AES_192_ECB ) {
```

Comment:
Minor style change (same as above):
```suggestion
            if( ( ret == MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED ||
                  ret == MBEDTLS_ERR_CIPHER_FEATURE_UNAVAILABLE ) &&
                  cipher_type == MBEDTLS_CIPHER_AES_192_ECB ) {
```